include/revng/Pypeline/Registrators/Container.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <functional>
+#include <map>
+#include <set>
+
+#include "nanobind/stl/set.h"
+#include "nanobind/stl/string.h"
+#include "nanobind/stl/string_view.h"
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/StringRef.h"
+
+#include "revng/Pypeline/Concepts.h"
+#include "revng/Pypeline/Registrators/Detail/PythonUtils.h"
+#include "revng/Pypeline/Registry.h"
+#include "revng/Support/Assert.h"
+
+namespace detail {
+
+template<typename T>
+struct ContainerSerdes {
```

Comment:
I'm not a fan of the "serdes" term. `ContainerIO`?

Reply:
Renamed to `ContainerIO`